clms-api/src/main/java/com/clms/api/assignments/AssignmentQuestion.java

```diff
@@ -43,6 +44,7 @@ public class AssignmentQuestion {
       isCorrect: boolean;
     }
      */
+    @Column(columnDefinition = "TEXT", length = 1024)
     @Convert(converter = AssignmentQuestionAnswerConverter.class)
     private List<AssignmentQuestionAnswer> answers;
 
```

Comment:
Changed asnwers and question to Text type in postgres, for some reason the ORM wouldnt allow the variable length without **length** defined in the column annotation.  It says 1024 length but I've been able to store atleast 30k characters